vars/deployToLambda.groovy

```diff
@@ -0,0 +1,35 @@
+def call(Map config) {
+    library("tdr-jenkinslib")
+    pipeline {
+        agent {
+            label "master"
+        }
+        parameters {
+            choice(name: "STAGE", choices: ["intg", "staging", "prod"], description: "The stage you are building the front end for")
```

Comment:
Probably want to change the description for the STAGE parameter

Reply:
Good point, thanks